base/accumulate.jl

```diff
@@ -85,6 +85,26 @@ julia> cumsum(a, dims=2)
  1  3   6
  4  9  15
 ```
+
+Note the important difference between `cumsum(A)` and `accumulate(+, A)` for
+arrays with small signed or unsigned integer eltype:
```

Comment:
"`" around `eltype` and a reference to the docs for that function?

Reply:
I do not think that a reference to the docs is appropriate in this case based on style found in [abstractarray.jl](https://github.com/JuliaLang/julia/blob/master/base/abstractarray.jl#L542).

Switched over to using `!!! note`